cranelift/filetests/filetests/isa/x64/band_not_bmi1.clif

```diff
@@ -8,14 +8,12 @@ block0(v0: i8, v1: i8):
     return v2
 }
 
-;   pushq   %rbp
-;   movq    %rsp, %rbp
-; block0:
-;   andn    %eax, %esi, %edi
-;   movq    %rbp, %rsp
-;   popq    %rbp
-;   ret
-
+;   pushq %rbp
+;   movq %rsp, %rbp
+;   andnl %edi, %esi, %eax
```

Comment:
Interesting, Capstone believes we printed the `andn` operands in reverse order. Could you double-check that and fix the pretty-printer if the disassembler is correct?

Reply:
I followed the intel manual when I added support for `andn`; I must have inadvertently implemented the intel formatting for the instruction :)

Reply:
If I switch back to intel syntax and regenerate this test output, I see the following:
```
;   push rbp                                                                         
;   mov rbp, rsp                                                                     
;   andn eax, esi, edi                                                               
;   mov rsp, rbp                                                                     
;   pop rbp                                                                          
;   ret   
```
I'll update the pretty-printer in a separate PR :+1:

Reply:
https://github.com/bytecodealliance/wasmtime/pull/5789